batch-add-recording-relationships.user.js

```diff
@@ -40,10 +36,13 @@
 // ==/License==
 
 const scr = document.createElement('script');
-scr.textContent = `(${batch_recording_rels})();`;
+scr.textContent = `(${batch_recording_rels})(${JSON.stringify(GM_info)});`;
 document.body.appendChild(scr);
 
-function batch_recording_rels() {
+function batch_recording_rels(gm_info) {
+    // eslint-disable-next-line no-useless-escape, prettier/prettier -- I really need to replace " by \"
+    let edit_note_signature = `\n\n\n'''${gm_info.script.name.replace(/"/g, '\"')}'''`;
+    edit_note_signature += ` ${gm_info.script.version}\n''Relate to %edit-mode%''`;
```

Comment:
```suggestion
    let edit_note_signature = `\n\n\n'''${gm_info.script.name}'''`;
    edit_note_signature += ` ${gm_info.script.version}\n''Relate to %edit-mode%''`;
```
The linter is right, the escape is indeed useless (`'\"' === '"'`) , in fact you don't even need to replace the quotes at all. The result is still the same for me:
```
'''MusicBrainz: Batch-add "performance of" relationships''' 2023.6.30.1538
''Relate to %edit-mode%''
```